src/Router.tsx

```diff
@@ -21,7 +21,7 @@ import Project from './components/Project';
 
 import LoginContext from './context/LoginContext';
 import { CHECK_USER_TOKEN } from './queries/AuthQueries';
-import { CheckUserToken } from './schemaTypes';
+import { ICheckUserToken } from './schemaTypes';
```

Comment:
Don't rename the interfaces in the schemaTypes file!
They will be overwritten every time we run codegen

Reply:
okay, will be fixed

Reply:
we don't have codegen in the front

Reply:
Yes we do. And we also have a script to launch it in the `package.json`:
```
"generate-gql-types": "apollo client:codegen --watch --includes='src/**/*.tsx' --endpoint=http://localhost:4000/graphql --target typescript --outputFlat src/schemaTypes.ts"
```